File: test/fixtures/php/j_patch_file_only.php
<?php
$ch = curl_init();
curl_setopt($ch, CURLOPT_URL, 'http://localhost:28139/patch');
curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
curl_setopt($ch, CURLOPT_CUSTOMREQUEST, 'PATCH');
curl_setopt($ch, CURLOPT_POSTFIELDS, [
    'file1' => new CURLFile('./test/fixtures/curl_commands/delete.sh'),
]);

$response = curl_exec($ch);

curl_close($ch);

File: test/fixtures/csharp/j_patch_file_only.cs
using System.Net.Http;

HttpClient client = new HttpClient();

HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Patch, "http://localhost:28139/patch");

MultipartFormDataContent content = new MultipartFormDataContent();
content.Add(new ByteArrayContent(File.ReadAllBytes("./test/fixtures/curl_commands/delete.sh")), "file1", Path.GetFileName("./test/fixtures/curl_commands/delete.sh"));
request.Content = content;

HttpResponseMessage response = await client.SendAsync(request);
response.EnsureSuccessStatusCode();
string responseBody = await response.Content.ReadAsStringAsync();
